spec/controllers/admin/api_controller_spec.rb

```diff
@@ -9,9 +9,9 @@
     let(:params) { {} }
 
     context "where there is no user or admin logged in" do
-      it "redirects to the homepage" do
-        get :index, params: params
-        it_redirects_to root_path
+      it "redirects to the homepage with a notice" do
+        get :index, params
```

Comment:
I think this should stay `params: params`. Just specifying `params` is the pre-Rails 5 syntax.

https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec#specify-managing-a-widget-with-rails-integration-methods